mindmeld/auto_annotator.py

```diff
@@ -463,14 +449,53 @@ def _load_model(model):
 
     @property
     def supported_entity_types(self):  # pylint: disable=W0236
-        """
+        """This function generates a list of supported entities for the given language.
+        These entities labels are mapped to MindMeld sys_entities.
+        The "misc" spacy entity is skipped since the category too broad to be
+        helpful in an application.
+
         Returns:
             supported_entity_types (list): List of supported entity types.
         """
-        return SPACY_ANNOTATOR_SUPPORTED_ENTITIES
+        spacy_supported_entities = [e.lower() for e in self.nlp.get_pipe("ner").labels]
```

Comment:
What exactly is `get_pipe `, it seems out of place to be a public method of the nlp object.

Reply:
It's a public method to get the component of your nlp. https://spacy.io/api/language#get_pipe